spark/src/test/scala/org/apache/spark/sql/delta/MergeIntoSQLSuite.scala

```diff
@@ -30,46 +30,13 @@ import org.apache.spark.sql.functions.udf
 import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.sql.types.{IntegerType, StructField, StructType}
 
-class MergeIntoSQLSuite extends MergeIntoSuiteBase  with MergeIntoNotMatchedBySourceSuite
+class MergeIntoSQLSuite extends MergeIntoSuiteBase
+  with MergeIntoSQLTestUtils  with MergeIntoNotMatchedBySourceSuite
```

Comment:
nit: 1 space indent here
```suggestion
  with MergeIntoSQLTestUtils with MergeIntoNotMatchedBySourceSuite
```